SimpCity/buildings/House.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SimpCity.buildings {
+    class House : CityGridBuilding {
+        public static string Name { get; } = "House";
+        /// <summary>
+        /// The 3-character code for the building
+        /// </summary>
+        public static string Code { get; } = "HSE";
+        public House(BuildingInfo info) : base(info) { }
+    }
+}
```

Comment:
As agreed with tech lead @jordie007, only Beach will ship in Sprint 1 while the rest will go with Sprint 2. Should remove this

Reply:
ok will delete it